apps/server/src/modules/files-storage/controller/test/files-storage-copy-files.e2e-spec.ts

```diff
@@ -25,6 +25,7 @@ import {
 import { Request } from 'express';
 import S3rver from 's3rver';
 import request from 'supertest';
+import { FileRecordParentType } from '../../entity';
```

Comment:
The order of imports seems not standard in some places like here. You can use VS Code's feature to organize imports. My shortcut is Alt+Shift+o (not sure if it's the default).

Reply:
Nothing changing on Alt+Shift+o

Reply:
I thought it would order our custom @s below the node modules but apparently not. Should have checked first myself...